verification/curator-service/api/src/controllers/access.ts

```diff
@@ -0,0 +1,12 @@
+import { intersection } from 'ramda';
+import { NextFunction } from 'express';
+
+const mustHaveRoles = (roles: Array<string>, req: any, res: any, next: NextFunction) => {
```

Comment:
:clown_face: I'm not happy with `req` and `res` being `any` type, but couldn't use the interface types defined by Express. For example, `req` interface doesn't have a `user` property. How do I typescript?